nltk/classify/decisiontree.py

```diff
@@ -339,7 +346,7 @@ def demo():
     classifier = names_demo(
         f, binary_names_demo_features  # DecisionTreeClassifier.train,
     )
-    print(classifier.pp(depth=7))
+    print(classifier.pretty_format(depth=7))
```

Comment:
👍

Reply:
`pp` was never even a function, not sure how it slipped through the CI at all =)

Reply:
It was the predecessor of `pretty_format` until 3.0.1